### ISSUE TITLE
Connect Import Data

### ISSUE THREAD

**Issue by TheGaHTK** [hzi-braunschweig/SORMAS-Project#4135]
Dear Sormas-Team,

if we have large amounts of peolople to get into our System the import function is very helpfull.

The Problem is we still have to connect these people manually to cases or events they belong to.

It would be great if we could add the case-/ event-ID within the importfile to connect them automaticly.

Thank you very much and stay healthy.

Best regards

Julian

**Comment by Marko-ilmkreis** [comment]
@TheGaHTK Welcome here. I work als for a GA. 

I found ths issue were the request was made before: #2077. I found also that in the data dictionary that under contacts is also listed the case reference as field "caze". 

@kwa20 @bernardsilenou Nevermind I tried to import a contact and mentioned a linked "caze" on the server https://test-de1.sormas.netzlink.com/. The result was an unexpected error.

> "Unerwarteter Fehler beim Import dieser Zeile. Bitte senden Sie Ihre Fehlerberichtsdatei an einen Administrator und entfernen Sie diese Zeile aus Ihrer Importdatei."

Here are the both files: 

[Test.zip](https://github.com/hzi-braunschweig/SORMAS-Project/files/5877955/Test.zip)

**Comment by MateStrysewske** [comment]
@TheGaHTK Hey! Please use the associated issue template when making a feature or change request or a bug report.

@Marko-ilmkreis It's actually a bug that "caze" is contained in the import template. Right now there's no functionality to link an imported entity to one that is already existing in the database. I agree that this is a very desirable feature though.

**Comment by Marko-ilmkreis** [comment]
@MateStrysewske No, its not in the template or I haven't seen it there :-) I found it in the data dictionary.

**Comment by MateStrysewske** [comment]
@Marko-ilmkreis Ah, I see, thanks!

**Comment by Marko-ilmkreis** [comment]
@MateStrysewske Maybe it could be an option to set the link between contacts and cases with the bulk editing in future? @TheGaHTK Would this an option for you?

**Comment by TheGaHTK** [comment]
Hello, 

@MateStrysewske  thank you for your quik response and sorry, next time i will try doing it correctly. 

@Marko-ilmkreis thank you too. For us it would be easyier to connect them right at the importfile, but if there would be another option will be fine too.

Thank you both and best regards
